### DISCUSSION TITLE
Geolocation capture field

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14158] [General]
i would like my staff to fill a lead form and also a possibility to capture his/her location when filling the form ? 
is it possible to create a field and capture their current live location? 

is this functionality also present in yetiforce ?
1 see maps n all ....

**Comment by mariuszkrzaczkowski** [comment]
give an example what exactly do you mean, what location are you writing about?

**Reply by rioshaz** [reply]
@mariuszkrzaczkowski 
suppose a user goes to a customer's office or any location and take their requirements in a lead form... so i want a field that should be able to capture his or her current physical location automatically,, so we know if the user really visited the client office or not...

auto-detect user location and fill that geolocation field or when a user submits a form the location can be seen in updates ....when he submitted a form..

**Reply by waran70** [reply]
https://www.w3schools.com/html/tryit.asp?filename=tryhtml5_geolocation

**Reply by mariuszkrzaczkowski** [reply]
there is no such thing ready,
doubts that everyone wants to allow the location,
you can create a form and use YetiREST to communicate with CRM

**Comment by rioshaz** [comment]
@mariuszkrzaczkowski  i think all companies that send sales rep to customers want to know if their sales rep really did go to customer locations or not...

@waran70 how to implement this HTML into a form in yetifroce ?  any idea.. all the datatype i check none of that can do this.. any help ?

**Reply by mariuszkrzaczkowski** [reply]
you need to create your own website which will communicate with CRM via API

**Reply by rioshaz** [reply]
yes i have my own website already... but if i have to create a complete form in my website and then it stores data in crm database... whats the point? 
cannot we create a geolocation datatype like we have  for email, mobile, text, image etc when adding new fields to modules ?

like u see here in erpnext : 

![erp](https://user-images.githubusercontent.com/54880467/106937410-b90c6880-6743-11eb-8b30-94e0a5dca9a8.png)

**Reply by vovpff** [reply]
@rioshaz what application you use as form platform?

**Reply by rioshaz** [reply]
> @rioshaz what application you use as form platform?
we dont use any application for form... we need this feature inside yetiforce to track sales rep.. if thet really visited client when filling the form or not....

The image i shared is from erpnext which has a datatype named geo-location  and when u insert this field to a form it can then capture user location....

any u help you can do buddy ?

**Comment by rioshaz** [comment]
can  i do this vai any workflow ?
